App Service Tutorial/README.md

```diff
@@ -0,0 +1,5 @@
+### App Service Tutorial
+
+The Using App Services Tutorial walks through the modification of the App built during the Introductory Tutorial.  In this tutorial we will learn how to use the App Service and Service Event Types to decouple the Client from the App.  Once that it done we will modify the implementation of the App to show that this can be done without having to make any Client changes.
```

Comment:
it -> is
```suggestion
The Using App Services Tutorial walks through the modification of the App built during the Introductory Tutorial.  In this tutorial we will learn how to use the App Service and Service Event Types to decouple the Client from the App.  Once that is done we will modify the implementation of the App to show that this can be done without having to make any Client changes.
```